plaidml2/bridge/keras/__init__.py

```diff
@@ -8,6 +8,7 @@
 import six
 
 import numpy as np
+import scipy.stats
```

Comment:
Does this come with numpy or will we need to eventually update our `not_setup.cfg` to include scipy? Not a problem yet because we don't generate wheels for plaidml2, but eventually...

Reply:
This has been in `plaidml/keras` for a long time. Probably it's coming from Keras.